src/api/goods.ts

```diff
@@ -10,10 +10,14 @@ export function getAll(): Promise<Good[]> {
 
 export const get5First = () => {
   return getAll()
-    .then(goods => goods); // sort and get the first 5
+    .then(goods => {
+      return goods
+        .sort((good1, good2) => good1.name.localeCompare(good2.name))
+        .slice(0, 5);
+    });
 };
 
 export const getRedGoods = () => {
   return getAll()
-    .then(goods => goods); // get only red
+    .then(goods => goods.filter(good => good.color === 'red'));
```

Comment:
you need just color here
```suggestion
    .then(goods => goods.filter(({ color })=> color === 'red'));
```